arch/lib/Makefile

```diff
@@ -155,7 +157,7 @@ define cmd_lib_bounds
 	 echo "#define GENERATED_BOUNDS_H"; \
 	 echo ""; \
 	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
-	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
+	 echo "#define MAX_NR_ZONES 3"; \
```

Comment:
> I see. Thanks. I made a new pull request. I have solved the coding style
> errors reported
> by checkpatch.pl script. This time the code should be cleaner.

Thanks.
I will review again your updates. 

> I think the next step is to use x86 headers instead of arm header. I will
> start to work on this.

great.

> I also have a long-term puzzle in my mind. Is it possible to make kswapd
> work in Lib OS.
> If kswapd can be ported to user land, all mm code can be ported without any
> problem.

Hmm, I would say it is possible. typical way to start with is do integrate it (kswapd) and will see what's missing and fix/glue them.